cvat-ui/src/components/annotation-page/canvas/views/canvas2d/canvas-wrapper.tsx

```diff
@@ -857,6 +857,8 @@ class CanvasWrapperComponent extends React.PureComponent<Props> {
             if (activatedState && activatedState.objectType !== ObjectType.TAG) {
                 canvasInstance.activate(activatedStateID, activatedAttributeID);
             }
+        } else if (workspace === Workspace.ATTRIBUTE_ANNOTATION) {
+            canvasInstance.fit();
```

Comment:
I would add an argument into the method: automatic=true. Thus we will not log the event.

Reply:
I already fixed that. Now event is logged only by explicit dblclick. See canvasView.ts